### PR TITLE
[Pods] DCOS-9960: Adding filter headline

### PR DESCRIPTION
**Pull request by wavesoft**
Adding the `Showing X of X Pods` headline on the pod instances talble.

image

### CODE REVIEW

File: src/js/components/PodInstancesView.js [hunk 1]
```diff
@@ -5,9 +5,11 @@ import PodInstancesTable from './PodInstancesTable';
 import PodInstanceStatus from '../constants/PodInstanceStatus';
 import PodUtil from '../utils/PodUtil';
 import PodViewFilter from './PodViewFilter';
+import FilterHeadline from './FilterHeadline';
```
Comment (orlandohohmeier) [inline src/js/components/PodInstancesView.js]: #nit please alphabetize imports

**Comment by orlandohohmeier** [comment]
LGTM 💯 ✨

File: src/js/components/PodInstancesView.js (hunk 1 above)
Comment (wavesoft) [inline src/js/components/PodInstancesView.js]: Fixed 👍

**Comment by orlandohohmeier** [comment]
LGTM 💯 ✨

**Pull request merged**